.pre-commit-config.yaml

```
@@ -0,0 +1,23 @@
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.3.0
+  hooks:
+    - id: check-added-large-files
+    - id: check-case-conflict
+    - id: check-executables-have-shebangs
+    - id: check-json
+    - id: check-vcs-permalinks
+    - id: check-yaml
+    - id: end-of-file-fixer
+    - id: trailing-whitespace
+- repo: https://github.com/pre-commit/mirrors-eslint
+  rev: v8.3.0  # Use the sha / tag you want to point at
+  hooks:
+  - id: eslint
+    files: \.[jt]sx?$  # *.js, *.jsx, *.ts and *.tsx
+    types: [file]
+- repo: git@github.com:Yelp/detect-secrets
```

```suggestion
- repo: https://github.com/Yelp/detect-secrets
```
https://results.pre-commit.ci/run/github/409311093/1638483045.Jzwqc6-7TkGXqxfNKxlQuw